src/components/SimplePieChart.js

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import * as d3 from 'd3';
+import PropTypes from 'prop-types';
+
+
+const SimplePiechart = ({ data, total }) => {
+  const height = 400;
+  const width = 400;
+
+  // this is so that the browser doesnt show the labels of each slice
+  // in the pie chart when the values of the piechart data array are all 0's
+  if (data.filter((ele) => ele.value === 0).length > 0) {
+    return <h1> Waiting for valid pie chart...</h1>;
```

Comment:
in general, we should only have 1 h1 element on the page. perhaps make this a p tag instead

Reply:
done